plugins/tag/commands

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
+source "$(dirname $0)/../common/functions"
+
+tag_image() {
+  docker tag -f "$1" "$2"
+}
+
+case "$1" in
+  tag:add)
+    [[ -z $2 ]] && echo "Please specify an app to run the command on" && exit 1
+    APP="$2"; IMAGE_TAG="$3"
+    verify_app_name "$APP"
+    verify_image "$IMAGE" || dokku_log_fail "App image ($IMAGE) not found"
+
+    tag_image "$IMAGE:latest" "$IMAGE:$IMAGE_TAG"
+    dokku_log_info2_quiet "Added $IMAGE_TAG tag to $IMAGE"
+    pluginhook tag-add $APP $IMAGE_TAG
+    ;;
+
+  tag:deploy)
+    [[ -z $2 ]] && echo "Please specify an app to run the command on" && exit 1
+    APP="$2"; IMAGE_TAG="$3"
+    verify_app_name "$APP"
+    verify_image "$IMAGE:$IMAGE_TAG" || dokku_log_fail "App image ($IMAGE:$IMAGE_TAG) not found"
+
+    release_and_deploy $APP $IMAGE_TAG
+    ;;
+
+  tag:list)
+    [[ -z $2 ]] && echo "Please specify an app to run the command on" && exit 1
+    APP="$2"
+    verify_app_name "$APP"
+
+    dokku_log_info2_quiet "Image tags for $IMAGE"
+    docker images "$IMAGE"
+    ;;
+
+  tag:rm)
+    [[ -z $2 ]] && echo "Please specify an app to run the command on" && exit 1
+    APP="$2"; IMAGE_TAG="$3"
+    verify_app_name "$2"
+
+    case "$IMAGE_TAG" in
+        latest)
+            dokku_log_fail "You can't remove internal dokku tag ($IMAGE_TAG) for $IMAGE"
+            ;;
+
+        *)
+            docker rmi "$IMAGE:$IMAGE_TAG"
+            ;;
+    esac
+    pluginhook tag-remove $APP $IMAGE_TAG
+    ;;
+
+  help | tag:help)
+    cat && cat<<EOF
+    tag:add <app> <tag>                             Add tag to latest running app image
```

Comment:
Can you update the with the new ", " convention instead of the hard white spacing? I'm guessing these lines might have been committed before that convention was widely used.